package.json

```diff
@@ -24,8 +24,7 @@
     "express": "^4.10.7",
     "express-http-proxy": "^0.6.0",
     "glob": "^4.3.2",
-    "lodash": "^2.4.1",
-    "lru-cache": "^2.5.0",
```

Comment:
Removed `lru-cache` because it was not being used and is already included by `on-core`